tools/ci/choose_workflows.py

```diff
@@ -57,7 +57,7 @@ def should_run_integration_tests(ctx: CIContext) -> bool:
         pr_keyword='[x] Run integration tests',
         pr_path_prefixes=[
             'Datadog/Example/',
-            'Tests/DatadogIntegrationTests/',
+            'IntegrationTests/IntegrationTests/',
             'Tests/DatadogBenchmarkTests/',
```

Comment:
🙌  It no longer makes sense to run integration tests on code changes in `Datadog/Example` or `Tests/DatadogBenchmarkTests/`.

Reply:
Indeed, good catch!